topology/sof-apl-tdf8532.m4

```diff
@@ -190,7 +190,7 @@ DAI_CONFIG(SSP, 2, 2, SSP2-Codec,
 	   SSP_CONFIG(DSP_B, SSP_CLOCK(mclk, 24576000, codec_mclk_in),
 		      SSP_CLOCK(bclk, 12288000, codec_slave),
 		      SSP_CLOCK(fsync, 48000, codec_slave),
-		      SSP_TDM(2, 32, 255, 255),
+		      SSP_TDM(8, 32, 255, 255),
```

Comment:
how about the masks? do we need enable all 8 bits for all slots of Tx and Rx?

Reply:
Sure, the TX and RX use the same clock and fs. When you set the RX with TDM8, you also should set the TX as the TDM8.